plugins/package-managers/bazel/src/main/kotlin/BazelModel.kt

```diff
@@ -31,8 +31,18 @@ private val json = Json { ignoreUnknownKeys = true }
  */
 @Serializable
 internal data class Lockfile(
-    val flags: Flags? = null
+    // The flags containing the registry URLs (Bazel < 7.2.0).
+    val flags: Flags? = null,
+
+    // The registry URLs of the project's dependencies packages  (Bazel >= 7.2.0).
+    val registryFileHashes: Map<String, String>? = null
```

Comment:
Consider using `emptyMap()` as the default.

Reply:
I am not sure about that. Checking if either of the value is `null` is the criteria to instantiate the registry service.
IMO introducing a non-`null` default value would increase the confusion.

Reply:
The lockfile seems to have the property: `lockFileVersion`.
I presume the version has been incremented with the introduction of `registryFileHashes`.
So, would a case distinction solely based on the `lockFileVersion` be more appropriate here ?

Reply:
I would rather that we not base ourselves on this `lockFileVersion` because:

1. The lock file has no public schema and can change in any sort of way with a new Bazel version
2. There is no mapping available between those `lockFileVersion` and Bazel versions (at least I didn't find any)